integration_tests/pages/page.ts

```diff
@@ -264,6 +277,75 @@ export default abstract class Page {
     })
   }
 
+  shouldContainRisksAndNeedsOasysMessage(): void {
+    cy.get('[data-testid="risks-and-needs-oasys-message"]').should(
+      'contain.text',
+      'Relevant sections from OASys to support the referral. The full Layer 3 assessment is available in OASys. Information is accurate at the time of the referral being submitted.',
+    )
+  }
+
+  shouldContainRisksAndNeedsSideNavigation(currentPath: string, referralId: Referral['id']): void {
+    const currentBasePath = currentPath.startsWith(assessPathBase.pattern)
+      ? assessPathBase.pattern
+      : referPathBase.pattern
+    const navigationItems = ShowRisksAndNeedsUtils.navigationItems(currentBasePath, referralId)
+
+    cy.get('.moj-side-navigation__item').each((navigationItemElement, navigationItemElementIndex) => {
+      const { href, text } = navigationItems[navigationItemElementIndex]
+
+      const { actual, expected } = Helpers.parseHtml(navigationItemElement, text)
+      expect(actual).to.equal(expected)
+
+      cy.wrap(navigationItemElement).within(() => {
+        cy.get('a').should('have.attr', 'href', href)
+
+        if (currentPath === href) {
+          cy.get('a').should('have.attr', 'aria-current', 'location')
+        } else {
+          cy.get('a').should('not.have.attr', 'aria-current', 'location')
+        }
+      })
+    })
+  }
+
+  shouldContainShowReferralSubHeading(): void {
+    cy.get('[data-testid="show-referral-sub-heading"]').should('contain.text', 'Risks and needs')
+  }
```

Comment:
Will this method be made more abstract later? If not, should the name reflect that this is specifically a risks and needs sub-heading?